aws/chamber/user.tf

```diff
@@ -7,7 +7,10 @@ module "chamber_user" {
   name          = "chamber"
   attributes    = ["codefresh"]
   kms_key_arn   = "${module.chamber_kms_key.key_arn}"
-  ssm_resources = ["${format("arn:aws:ssm:%s:%s:parameter/kops/*", var.region, var.account_id)}"]
+  ssm_resources = [
+    "${format("arn:aws:ssm:%s:%s:parameter/kops/*", var.region, var.account_id)}",
+    "${format("arn:aws:ssm:%s:%s:parameter/app/*", var.region, var.account_id)}"
```

Comment:
app is more symbolic. I think each app should get it's own namespace. I think we should change this to a `formatlist` and introduce a variable for the chamber services. Then we can pass it `kops`, `foo` and `bar`. 

Also, Chamber with codefresh should be deprecated due to secrets getting exposed by helm values.